File: Javascript/1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let values = lines.shift().split(' ');
let x1 = Number(values[0]), y1 = Number(values[1]);
values = lines.shift().split(' ');
let x2 = Number(values[0]), y2 = Number(values[1]);

console.log(Number( Math.sqrt( Math.pow((x1 - x2), 2) + Math.pow((y1 - y2), 2) ).toFixed(4) ));

File: C/1015.c
#include <stdio.h>
 
int main() {
 
    double X1,X2,Y1,Y2;
    scanf("%lf %lf %lf %lf", &X1, &Y1, &X2, &Y2);
    printf("%.4f\n", sqrt( pow((X2-X1),2) + pow((Y2-Y1),2) ));
 
    return 0;
}